_includes/asg/ebs-template.md

```diff
@@ -10,21 +14,26 @@ Ensure that these EBS volumes are created in the same region as the auto scaling
 
 Record the EBS volume ID (e.g. _vol-04e2283f1925ec9ee_), this will be passed in to PX as a parameter as a storage device.
 
-**For PX version > 1.3**
+### Using a template specification
 
-A set of EBS volume templates can be specified in the following comma-separated format
+For PX 1.3 and higher, you can specify a template spec which will be used by Portworx to create new EBS volumes.
 
+The spec follows the following format:
+```
+"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>"
 ```
-"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>","type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>"
 
-For ex:
+* __type__: Following two types are supported
+    * _gp2_
+    * _io1_ (For io1 volumes specifying the iops value is mandatory.)
+* __size__: This is the size of the EBS volume in GB
+* __iops__: This is the required IOs per second from the created EBS volume.
```

Comment:
nit: This is the required IOs per second from the EBS volume.

Reply:
done